app/assets/javascripts/views/story_view.js

```diff
@@ -226,6 +227,27 @@ module.exports = FormView.extend({
     }
   },
 
+  renderDate:function() {
```

Comment:
@dmitryrck That's because the template from a normal story differs from a release story.
Check it out:
https://github.com/Codeminer42/cm42-central/blob/72842e0bd6ae68bea7e6e1e7deb1e0447a60472e/app/assets/javascripts/views/story_view.js#L441